src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -849,41 +849,44 @@ class C
-                Diagnostic(ErrorCode.ERR_MissingPredefinedMember, "(int, string a)").WithArguments("System.ValueTuple`2", "Item2").WithLocation(6, 9),
-                // (6,29): error CS8204: Tuple member names must all be provided, if any one is provided.
-                //         (int, string a) x = (b: 1, "hello", 2);
-                Diagnostic(ErrorCode.ERR_TupleExplicitNamesOnAllMembersOrNone, @"(b: 1, ""hello"", 2)").WithLocation(6, 29),
-                // (6,29): error CS0518: Predefined type 'System.ValueTuple`3' is not defined or imported
-                //         (int, string a) x = (b: 1, "hello", 2);
-                Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound, @"(b: 1, ""hello"", 2)").WithArguments("System.ValueTuple`3").WithLocation(6, 29),
-                // (6,29): error CS0656: Missing compiler required member 'System.ValueTuple`3.Item1'
-                //         (int, string a) x = (b: 1, "hello", 2);
-                Diagnostic(ErrorCode.ERR_MissingPredefinedMember, @"(b: 1, ""hello"", 2)").WithArguments("System.ValueTuple`3", "Item1").WithLocation(6, 29),
-                // (6,29): error CS0656: Missing compiler required member 'System.ValueTuple`3.Item2'
-                //         (int, string a) x = (b: 1, "hello", 2);
-                Diagnostic(ErrorCode.ERR_MissingPredefinedMember, @"(b: 1, ""hello"", 2)").WithArguments("System.ValueTuple`3", "Item2").WithLocation(6, 29),
-                // (6,29): error CS0656: Missing compiler required member 'System.ValueTuple`3.Item3'
-                //         (int, string a) x = (b: 1, "hello", 2);
-                Diagnostic(ErrorCode.ERR_MissingPredefinedMember, @"(b: 1, ""hello"", 2)").WithArguments("System.ValueTuple`3", "Item3").WithLocation(6, 29)
-                );
+                //         (int, string) x = (1, "hello");
+                Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound, "(int, string)").WithArguments("System.ValueTuple`2").WithLocation(6, 9),
+                // (6,27): error CS0518: Predefined type 'System.ValueTuple`2' is not defined or imported
+                //         (int, string) x = (1, "hello");
+                Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound, @"(1, ""hello"")").WithArguments("System.ValueTuple`2").WithLocation(6, 27)
+                               );
+        }
+
+        [Fact]
+        public void TupleUsageWithMissingTupleMembers()
+        {
+            var source = @"
+namespace System
+{
+    public struct ValueTuple<T1, T2> { }
+}
+
+class C
+{
+    static void Main()
+    {
+        (int, int) x = (1, 2);
+    }
+}
+";
+            var comp = CreateCompilationWithMscorlib(source);
+            comp.VerifyEmitDiagnostics(
+                // (11,24): error CS8205: Member 'System_ValueTuple_T1_T2__ctor' was not found on type 'System.ValueTuple<T1, T2>' from assembly '__assembly_guid__, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.
```

Comment:
We definitely shouldn't include enum member name 'System_ValueTuple_T1_T2__ctor' into the message. That should be the name of the member we are looking for.